File: 250331/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    int gender, age;
    cin >> gender;
    cin >> age;

    if(gender == 0)
    {
        if(age >= 20)
            cout << "MAN" << endl;
        else
            cout <<"BOY" << endl;
    }
    else
        if (age >= 20)
            cout << "WOMAN" << endl;
        else
            cout << "GIRL" << endl;
            
    return 0;
}

File: 260114/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int sex = in.nextInt();
        int age = in.nextInt();

        if(sex == 0 && age >= 19){
            System.out.println("MAN");
        }
        else if(sex == 1 && age >= 19){
            System.out.println("WOMAN");
        }
        else if(sex == 0 && age < 19){
            System.out.println("BOY");
        }
        else{
            System.out.println("GIRL");
        }
    }
}